android/app/src/main/java/com/mulkkam/ui/history/HistoryViewModel.kt

```diff
@@ -99,13 +99,33 @@ class HistoryViewModel : ViewModel() {
             val result = RepositoryInjection.intakeRepository.deleteIntakeHistoryDetails(history.id)
             runCatching {
                 result.getOrError()
+                updateIntakeHistoriesAfterDeletion(history)
                 _deleteSuccess.value = true
             }.onFailure {
                 // TODO : 에러 처리
             }
         }
     }
 
+    private fun updateIntakeHistoriesAfterDeletion(history: IntakeHistory) {
+        val currentDailySummary = _dailyIntakeHistories.value ?: return
+
+        val newDailySummary = currentDailySummary.afterDeleteHistory(history)
```

Comment:
```suggestion
       val newDailySummary = _dailyIntakeHistories.value?.afterDeleteHistory(history) ?: return

```

p3) 이렇게 변수를 줄여 볼 수도 있겠네요 !

Reply:
p3) 아래에서 `currentDailySummary` 함수가 필요하여 분리 해두었습니다 :)